inference-engine/src/cldnn_engine/cldnn_config.cpp

```diff
@@ -95,7 +119,27 @@ void Config::UpdateFromMap(const std::map<std::string, std::string>& configMap)
                     queuePriority = cldnn::priority_mode_types::high;
                     break;
                 default:
-                    IE_THROW(ParameterMismatch) << "Unsupported queue priority value: " << uVal;
+                    IE_THROW(ParameterMismatch) << "Unsupported queue priority value: " << quque_priority;
+            }
+            switch (tbb_affinity) {
+                case 0: // default
+                    if (getAvailableCoresTypes().size() > 1)
+                        cpu_core_type = IStreamsExecutor::Config::BIG;
```

Comment:
as we discusssed offline, lets change this to ANY always

Reply:
Fixed it